lib/web_ui/lib/src/engine/dom_renderer.dart

```diff
@@ -814,6 +711,119 @@ $_glassPaneTagName * {
   void debugPlainTextLayout() => _debugFrameStatistics!.plainTextLayouts++;
 }
 
+// Applies the required global CSS to an incoming [html.CssStyleSheet] `sheet`.
+void applyGlobalCssRulesToSheet(html.CssStyleSheet sheet, {
+  required BrowserEngine browserEngine,
+  required bool hasAutofillOverlay,
+  String glassPaneTagName = DomRenderer._glassPaneTagName,
+}) {
+  final bool isWebKit = browserEngine == BrowserEngine.webkit;
+  final bool isFirefox = browserEngine == BrowserEngine.firefox;
+  // TODO(butterfly): use more efficient CSS selectors; descendant selectors
```

Comment:
Butterfly? 

![Are you from the past?](https://i.imgur.com/vrsLBmp.gif)

Reply:
🤣